### DISCUSSION TITLE
minus amount show front of " number of blocks left "

### DISCUSSION THREAD

**Discussion by ridmapathum** [pactus-project/pactus#611] [Q&A] [answered]
my system time was wrong and i fix it . then the error was fixed . if any one get this error check your system time .
thank you

**Comment by kehiy** [comment]
Hi @ridmapathum,

Probably your issue will be fixed in :
- https://github.com/pactus-project/PIPs/pull/21

**Reply by b00f** [reply, marked as answer]
It fixed at #731
